base/challenge-skeleton/README.md

```diff
@@ -1,52 +1,61 @@
-# Quickstart guide to writing a challenge:
+# Quickstart guide to writing a challenge
 
-* A docker image will be built from `challenge/image`. For the simplest challenges, replacing `challenge/image/chal` will be enough.
-* Edit `challenge/image/Dockerfile` to change the commandline or which files should get included.
+The basic steps when preparing a challenge are:
+
+* A Docker image is built from `challenge/image`. For the simplest challenges, replacing `challenge/image/chal` is sufficient.
+* Edit `challenge/image/Dockerfile` to change the commandline or the files you want to include.
 * Try the challenge locally with `make test-docker`.
-* If you have a cluster ready, deploy the challenge with `make start`.
-  * To access it, create a port forward with `make port-forward` and connect via `nc localhost PORT` using the printed port.
+* If you have prepared a cluster, deploy the challenge with `make start`.
+  * To access the challenge, create a port forward with `make port-forward` and connect to it via `nc localhost PORT` using the printed port.
```

Comment:
Port forwarding only works inside a cluster, so maybe we should just use instructions here that work for local dev?

Reply:
that's why it's indented, you think that's not clear?

Reply:
Ok, I missed the indentation :-)